src/js/containers/MinifyCard/MinifyCard.js

```diff
@@ -18,6 +18,17 @@ class MinifyCard extends Component {
         };
     }
 
+    componentWillMount() {
```

Comment:
Linter was complaining about modifying state without calling `setState()`.  Moved it into `componentWillMount` because it will execute when props change and is where this *should* have gone originally but I didn't know react that well when I originally wrote this.